### DISCUSSION TITLE
Release of 1.6 with support of CAPI 1.3

### DISCUSSION THREAD

**Discussion by farodin91** [kubernetes-sigs/cluster-api-provider-vsphere#1793] [General]
It would be really, if 1.6 gets release any time soon. 

Can I help some ways?

**Comment by srm09** [comment]
It is a valid ask, I wanted to fix this one https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/1788 before cutting the next minor release

**Reply by srm09** [reply]
This should fix the failing e2e tests https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/1794
PTAL @farodin91
